website/docs/whats-new/release-notes/partials/components.md

```diff
@@ -0,0 +1,200 @@
+
+- `Pagination` bugfix `aria-label` on the component. [(#1627)](https://github.com/hashicorp/design-system/pull/1627) Thanks [@MelSumner](https://github.com/MelSumner)! 
+
+- `Dropdown` changed `@height` property to use `max-height` instead of a fixed height.[(#1635)](https://github.com/hashicorp/design-system/pull/1635) Thanks [@KristinLBradley](https://github.com/KristinLBradley)! 
+
+## 2.12.0
+
+**Month Day, Year**
+
+- `Design tokens` added color tokens for “Vault Secrets” product [(#1640)](https://github.com/hashicorp/design-system/pull/1640) Thanks [@didoo](https://github.com/didoo)! 
+
+- `IconTile` updated component to include `vault-secrets` product option.
+
+- `Text` - Removed leftover `console.log` from code. [(#1615)](https://github.com/hashicorp/design-system/pull/1615) Thanks [@didoo](https://github.com/didoo)! 
+
+- `Dropdown` refactored the layout of the checkbox and radio inputs to make the gap between the inputs and the associated text, as well as the icon and count, clickable. [(#1618)](https://github.com/hashicorp/design-system/pull/1618) Thanks [@KristinLBradley](https://github.com/KristinLBradley)!
+
+- `Hds::Link::Standalone` change font-weight from 500 to 400 to match font-weight of `Hds::Button`. [(#1617)](https://github.com/hashicorp/design-system/pull/1617) Thanks [@KristinLBradley](https://github.com/KristinLBradley)!
+
+- `Stepper` removed some CSS declarations that were not used/applied. [(#1628)](https://github.com/hashicorp/design-system/pull/1628) Thanks [@didoo](https://github.com/didoo)!
+
+- 🔄 Updated dependencies:
+    - @hashicorp/design-system-tokens@1.8.0
+    - @hashicorp/ember-flight-icons@3.1.2
+
+## 2.11.0
+
+**Month Day, Year**
+
+- `Hds::Text` added new `Text` component. [(#1490)](https://github.com/hashicorp/design-system/pull/1490) Thanks [@didoo](https://github.com/didoo)!
+
+- `Hds::Form::MaskedInput` add `hasCopyButton` argument. [(#1587)](https://github.com/hashicorp/design-system/pull/15807) Thanks [@alex-ju](https://github.com/alex-ju)!
+
+- `Form::Indicator` removed aria-hidden from the "optional" span [(#1577)](https://github.com/hashicorp/design-system/pull/1577) Thanks [@DingoEatingFuzz](https://github.com/DingoEatingFuzz)!
+
+- `ember-named-blocks-polyfill` removed as all consumers of HDS are on Ember 3.25 or later now. This can be installed locally if it is still needed. [(#1606)](https://github.com/hashicorp/design-system/pull/1577) Thanks [@Dhaulagiri](https://github.com/didoo)! 
+
+- `Alert`, `Toast` fixed an issue with anchor tag color styles within Description that had been overriding `Hds::Link color`; changed the default color for HTML links within Description to "neutral" to better align with existing guidance for links in the actions and improve accessible contrast. [(#1576)](https://github.com/hashicorp/design-system/pull/1577) Thanks [@KristinLBradley](https://github.com/KristinLBradley)! 
+
+- 🔄 Updated dependencies:
+    - @hashicorp/ember-flight-icons@3.1.1
```

Comment:
```suggestion
**August 31, 2023**

- `Hds::Text` - added new `Text` component. [#1490](https://github.com/hashicorp/design-system/pull/1490) Thanks [@didoo](https://github.com/didoo)!

- `Hds::Form::MaskedInput` - added `hasCopyButton` argument. [#1587](https://github.com/hashicorp/design-system/pull/15807) Thanks [@alex-ju](https://github.com/alex-ju)!

- `Form::Indicator` - removed aria-hidden from the "optional" span [#1577](https://github.com/hashicorp/design-system/pull/1577) Thanks [@DingoEatingFuzz](https://github.com/DingoEatingFuzz)!

- `ember-named-blocks-polyfill` - removed as all consumers of HDS are on Ember 3.25 or later now. This can be installed locally if it is still needed. [#1606](https://github.com/hashicorp/design-system/pull/1577) Thanks [@Dhaulagiri](https://github.com/didoo)! 

- `Alert` and `Toast` - fixed the following issues with the links in the components: 
 
    - fixed an issue with the anchor tag color styles within Description that had been overriding `Hds::Link color`
    - changed the default color for HTML links within the Description to "neutral" to better align with existing guidance for links in the actions and improve accessible contrast. 

[#1576](https://github.com/hashicorp/design-system/pull/1577) Thanks [@KristinLBradley](https://github.com/KristinLBradley)! 

- 🔄 Updated dependencies:
    - @hashicorp/ember-flight-icons@3.1.1
```